cli/src/daemon.rs

```diff
@@ -63,6 +63,9 @@ struct MaintenanceMetrics {
 
     /// Number of times we performed `Unstake` on an active validator for balancing purposes.
     transactions_unstake_from_active_validator: u64,
+
+    /// Number of times we performed `SellRewards` on the Anker instance.
+    transactions_sell_rewards: u64,
```

Comment:
This also needs a few new lines below in `write_prometheus`, but I fixed that in #481 already.